src/sentry/event_manager.py

```diff
@@ -662,6 +664,22 @@ def _pull_out_data(jobs, projects):
         )
 
 
+def _is_commit_sha(version: str):
+    return re.match(r"[0-9a-f]{40}", version) is not None
+
+
+def _associate_commits_with_release(release: Release):
+    previous_release = release.previous_release
+    fetch_commits.apply_async(
```

Comment:
I'm rather confused with this.

Unless `refs` is to be passed this call should not do anything useful.

Reply:
Yeah, it's a placeholder for now. I'm not sure how to fill out `ref` in this context